Adding sc4.8.0-introduced CLI flags description (#1231)
saucelabs/sauce-docs: docs/dev/cli/sauce-connect-proxy.md

## External Proxy Configuration

---
### `--autodetect`

> __Description__: Enables the auto-detection of system proxy settings. Inverse of [`--no-autodetect`](#--no-autodetect). Default: true. See also [Automatic Proxy Auto-Configuration](/secure-connections/sauce-connect/setup-configuration/additional-proxies#proxy-auto-configuration-automatic).<br/>

Reviewer: ```suggestion
__Description__: Enables the auto-detection of system proxy settings. Inverse of [`--no-autodetect`](#--no-autodetect). Default: `true`. See also [Automatic Proxy Auto-Configuration](/secure-connections/sauce-connect/setup-configuration/additional-proxies#proxy-auto-configuration-automatic).<br/>
```

__Shorthand__: n/a

---
### `--output-format`
<p><small>| OPTIONAL | STRING |</small></p>

> __Description__: Console output format configuring either "pretty" output incuding different fonts and graphics or text-only. One of: pretty, text. Default: pretty.<br/>

Reviewer: ```suggestion
__Description__: Specifies console output format. You can configure either `pretty` output, which will display various fonts and graphics, or `text` (text only). Default: `pretty`.<br/>
```

__Shorthand__: n/a

---
### `--experimental`
<p><small>| OPTIONAL | STRING |</small></p>

> __Description__: Enable or disable experimental features. This flag allows controlled replacement of the components. It should only be used if the default feature configuration exhibit undesired behavior.<br/>

Reviewer: ```suggestion
__Description__: Enable or disable experimental features. This flag allows controlled replacement of the components. It should only be used if the default feature configuration exhibits undesired behavior.<br/>
```
